docs/source/contact.rst

```diff
@@ -0,0 +1,7 @@
+.. _contact:
+
+Contact Us
+==========
+
+If you are looking to contact us, including looking for support, please see our
```

Comment:
As said on README, since PyJNIus can be also used without knowing what Kivy is, that could be confusing, maybe is better to specify it?

E.g.

PyJNIus is part of the [Kivy](link) ecosystem, and if you're looking to contact us ...